src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/azclishell/az_completer.py

```diff
@@ -205,8 +205,11 @@ def mute_parse_args(self, text):
         AzCliCommandParser.error = error_pass
         AzCliCommandParser._check_value = _check_value_muted
 
-        parse_args = self.argsfinder.get_parsed_args(
-            parse_quotes(text, quotes=False, string=False))
+        try:
+            parse_args = self.argsfinder.get_parsed_args(
+                parse_quotes(text, quotes=False, string=False))
+        except Exception:  # pylint: disable=broad-except
```

Comment:
What kind the exception will actually be thrown here?

Reply:
From what I can see, none. argparse will call `error()` (which is also patched) with any parser related errors. 
This is more of a safety assurance change- we don't want the patched methods to persist anywhere beyond this scope.

Reply:
Add comments in code.